corehq/apps/hqcase/utils.py

```diff
@@ -221,7 +221,7 @@ def bulk_update_cases(domain, case_changes, device_id):
         # Ensure the XML is formatted properly
         # An exception is raised if not
         case_block = ElementTree.tostring(case_block.as_xml())
-        case_blocks.append(case_block)
+        case_blocks.append(case_block.as_text())
```

Comment:
@snopoke 
this fails with
```
ipdb> case_block
b'<case case_id="122" date_modified="2019-10-18T15:51:10.465704Z" user_id="system" xmlns="http://commcarehq.org/case/transaction/v2"><update><name>motu</name></update></case>'
ipdb> case_block.as_text()
*** AttributeError: 'bytes' object has no attribute 'as_text'
ipdb> case_block.as_xml
```
works if i remove the `ElementTree` call and just do `case_blocks.append(case_block.as_text())` but then the comment above says `Ensure the XML is formatted properly` which would be skipped if we do just `as_text`.
should we reconsider using 
`case_block = ElementTree.tostring(case_block.as_xml()).decode('utf-8')`?

Reply:
The intention was this:
```diff
- case_block = ElementTree.tostring(case_block.as_xml())
- case_blocks.append(case_block)
+ case_blocks.append(case_block.as_text())
```

If you look at the implementation of `case_block.as_text()` you'll see that it already does the decoding and is equivalent to `ElementTree.tostring(case_block.as_xml()).decode('utf-8')`

Reply:
thank you @snopoke 
Updated, https://github.com/dimagi/commcare-hq/pull/25624/commits/7c8d5f5b47fbaa676dc5bcaac087de5ded344661